hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java

```diff
@@ -134,7 +134,6 @@ public QuorumJournalManager(Configuration conf,
 
   }
 
-  
```

Comment:
avoid unnecessary changes

Reply:
Thanks, I rolled it back in the latest commit.